src/Compilers/CSharp/Portable/FlowAnalysis/PreciseAbstractFlowPass.cs

```diff
@@ -1293,6 +1293,11 @@ public override BoundNode VisitLiteral(BoundLiteral node)
             return null;
         }
 
+        public override BoundNode VisitMethodToken(BoundMethodToken node)
+        {
+            return null;
```

Comment:
If performance is a concern, which is premature at this point, there are many different ways to mitigate it (for example, using specialized subclass of LocalRewriter when instrumentation is needed). However, I would be surprised if a check on a boolean field or a null check on a field (if we would pass an optional "statement instrumenter" to the LocalRewriter) would create any noticeable performance impact.